website/pages/blog/judging-open-source-by-github-stars.mdx

```diff
@@ -0,0 +1,254 @@
+---
+title: Judging Open Source by Github Stars
+tags: [open-source]
+authors: yassin
+date: 2023-05-09
+description: Exposing the dark side of GitHub and open source projects
+image: /blog-assets/judging-open-source-by-github-stars/cover.png
+thumbnail: /blog-assets/judging-open-source-by-github-stars/thumbnail.png
+---
+
+The best and most obvious way to judge an open-source project is to look at the code but this
+can be kind of tedious and sometimes you don't like what you see there, so an alternative that we
+have all naturally developed on our own or have been advised to, is to see how many people have
+starred a project, and then pick the one with the most stars.
```

Comment:
```suggestion
How to judge an open-source project?

The best way to judge an open-source project is by looking directly at the code, but this can be kind of tedious, and sometimes you don't like what you see there.

So, an alternative that we have all naturally developed on our own or have been advised to, is to see how many people have starred a project, and then pick the one with the most stars.
```